src/test/java/com/epam/tc/hw1/AbstractTest.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.tc.hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.asserts.SoftAssert;
+
+
+public class AbstractTest {
+    protected Calculator calculator;
+    protected SoftAssert softAssert;
+
+    @BeforeMethod
+    public void before() {
+        calculator = new Calculator();
+        softAssert = new SoftAssert();
+    }
+
+    @AfterMethod
+    public void after() {
+        calculator = null;
+        softAssert = null;
+    }
```

Comment:
Do you really need softAssert in all your classes? i think that SoftAssert should be method variable.